### ISSUE TITLE
cars disapearing :(

### ISSUE THREAD

**Issue by daciansolgen3** [MAFINS/MenyooSP#117]
when i spawn cars from dlc they all disapears :( could you take a look and see if there is possible to fix it??

**Comment by KillerBoy001** [comment]
yeah online only cars despawn if he does not use SET_ENTITY_AS_MISSION_ENTITY on them , so yes it fixable

**Comment by Pombart** [comment]
Could i fix that myself? Where would i have to put that?

**Comment by lostsol87** [comment]
how can we fix this? it happens to me to

**Comment by mamadrol007** [comment]
For everyone who still has this problem: instal " Add-On Vehicle Spawner " (also on the GTA5 Mods website).
its an easy instal and will alow you to spawn all cars also gta online cars. So far everyting works with me and no cars despawn.
Its a little dificult to navigate but it works. you can use this mod to spawn the cars and you can still use menyoo to customize them.
( sorry for bad english)

I hope it works fot you guys, enjoy !
